src/main/java/com/dmytrohuk/weborganizer/users/User.java

```diff
@@ -75,12 +76,12 @@ public void setFirstName(String firstName) {
         this.firstName = firstName;
     }
 
-    public String getLastName() {
-        return lastName;
+    public String getSurname() {
```

Comment:
Replace by corresponding `@Getter` and `@Setter` annotations (better above whole class)